components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/password-reset-complete.jsp

```diff
@@ -51,7 +58,12 @@
         property.setKey("callback");
         property.setValue(URLEncoder.encode(callback, "UTF-8"));
         properties.add(property);
-        
+
+        Property userPortalURLProperty = new Property();
+        userPortalURLProperty.setKey("isUserPortalURL");
+        userPortalURLProperty.setValue(String.valueOf(isUserPortalURL));
```

Comment:
shall we use boolean.parseboolean(isUserPortalURL) here?

Reply:
isUserPortalURL is already a boolean value. Why do you expect it to be parsed into a boolean value again? Hence userPortalURLProperty.setValue() method accepts only String value, I have wrapped the isUserPortalURL variable into a String.